tests/Unit/DeliveryResult/Factory/DeliveryResultFilterFactoryTest.php

```diff
@@ -45,7 +47,7 @@ public function setUp(): void
 
         $this->subject = new DeliveryResultFilterFactory();
         $this->subject->setServiceLocator(
-            $this->getServiceLocatorMock(
+            $this->getServiceManagerMock(
```

Comment:
Why do you need to change this?

Reply:
Because getServiceLocatorMock was deprecated 
https://github.com/oat-sa/generis/blob/master/test/TestCase.php#L41